extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -348,6 +356,27 @@ export class AmpStoryPage extends AMP.BaseElement {
     this.setPageDescription_();
   }
 
+  /** @private */
+  maybeSetPreviewDuration_() {
+    if (this.storeService_.get(StateProperty.PREVIEW_MODE)) {
+      const videos = this.getAllVideos_();
+
+      const autoAdvanceAttr =
+        videos.length > 0
+          ? VIDEO_PREVIEW_AUTO_ADVANCE_DURATION
+          : DEFAULT_PREVIEW_AUTO_ADVANCE_DURATION;
+
+      addAttributesToElement(
+        this.element,
+        dict({
+          'auto-advance-after': autoAdvanceAttr,
+        })
+      );
+
+      this.advancement_ = AdvancementConfig.forElement(this.win, this.element);
```

Comment:
Side note: The old advancement config should be deleted by the garbage collector, but because it holds references on existing elements (like a media element) it likely won't be evicted. Why it doesn't matter so much since we're not running this code thousands of times, it's the #1 recipe for memory leaks

Reply:
Good point 👍